h/views/oidc.py

```diff
@@ -128,7 +128,8 @@ def notfound(self):
 
 @view_defaults(request_method="GET", route_name="oidc.redirect.orcid")
 class ORCIDRedirectViews:
-    def __init__(self, request: Request) -> None:
+    def __init__(self, context, request: Request) -> None:
+        self._context = context
```

Comment:
When an exception view is called what happens is:

1. Pyramid constructs _another_ instance of `ORCIDRedirectViews`: it may have already constructed a first `ORCIDRedirectViews` instance in order to call the view that raised the exception. It now constructs a second, separate instance in order to call the exception view.
2. When constructing the second, exception view instance of `ORCIDRedirectViews` Pyramid passes the exception that was raised as `context`. The same thing happens when an exception view is just a function rather than a class method: the exception is passed to the function as the `context` argument.